diff --git a/packages/server/service/index.js b/packages/server/service/index.js
index 4a1f568..f46c8c6 100644
--- a/packages/server/service/index.js
+++ b/packages/server/service/index.js
@@ -5,14 +5,14 @@ const { Database, PlcServer } = require('..')
 
 const main = async () => {
   const version = process.env.PLC_VERSION
-  const dbCreds = JSON.parse(process.env.DB_CREDS_JSON)
+  const dbUrl = process.env.DB_URL
   const dbSchema = process.env.DB_SCHEMA || undefined
   const enableMigrations = process.env.ENABLE_MIGRATIONS === 'true'
   if (enableMigrations) {
-    const dbMigrateCreds = JSON.parse(process.env.DB_MIGRATE_CREDS_JSON)
+    const dbMigUrl = process.env.DB_MIGRATE_URL
     // Migrate using credentialed user
     const migrateDb = Database.postgres({
-      url: pgUrl(dbMigrateCreds),
+      url: dbMigUrl,
       schema: dbSchema,
     })
     await migrateDb.migrateToLatestOrThrow()
@@ -23,7 +23,7 @@ const main = async () => {
   const dbPoolIdleTimeoutMs = parseMaybeInt(process.env.DB_POOL_IDLE_TIMEOUT_MS)
   // Use lower-credentialed user to run the app
   const db = Database.postgres({
-    url: pgUrl(dbCreds),
+    url: dbUrl,
     schema: dbSchema,
     poolSize: dbPoolSize,
     poolMaxUses: dbPoolMaxUses,
@@ -40,11 +40,6 @@ const main = async () => {
   })
 }
 
-const pgUrl = ({ username = "postgres", password = "postgres", host = "localhost", port = "5432", database = "postgres", sslmode }) => {
-  const enc = encodeURIComponent
-  return `postgresql://${username}:${enc(password)}@${host}:${port}/${database}${sslmode ? `?sslmode=${enc(sslmode)}` : ''}`
-}
-
 const parseMaybeInt = (str) => {
   return str ? parseInt(str, 10) : undefined
 }
